nexus/src/main/kotlin/com/vanniktech/maven/publish/nexus/Nexus.kt

```diff
@@ -274,6 +280,36 @@ class Nexus(
     dropStagingRepository(stagingRepository.repositoryId)
   }
 
+  /** A simple logger interface that can start, complete, and report intermediate progress. */
+  interface Logger {
+    fun start(description: String, status: String)
+    fun progress(status: String, failing: Boolean = false)
+    fun completed(status: String, failed: Boolean)
+
+    /** A system logger that writes to [System.out] or [System.err]. */
+    object SystemLogger : Logger {
+      override fun start(description: String, status: String) {
+        println("$description: $status")
+      }
+      override fun progress(status: String, failing: Boolean) {
+        if (failing) {
+          System.err.print("\r$status")
+          System.err.flush()
+        } else {
+          print("\r$status")
+          System.out.flush()
+        }
+      }
+      override fun completed(status: String, failed: Boolean) {
+        if (failed) {
+          System.err.println("Completed with errors: $status")
+        } else {
+          println("Completed: $status")
```

Comment:
Can you try adding a \n at the beginning of both of these? That should fix the missing new line for the completed releasing message at the end, it's probably coming from the progress updates never ending in a new line

Reply:
[b876a23](https://github.com/vanniktech/gradle-maven-publish-plugin/pull/525/commits/b876a23bf04fb4412aba4191189a1583868dbdbc)